### PR TITLE
Nodejs auth example

### PR DESCRIPTION
**Pull request by jona-termly**

### CODE REVIEW

File: auth-examples/.DS_Store
Comment (Gidgidonihah) [inline auth-examples/.DS_Store:1]: Let's not commit this or any other `.DS_Store` files. Might be easier to just move that line from the `auth-examples/nodejs/.gitignore` to `.gitignore` and remove them.

File: auth-examples/nodejs/server.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+app.get('/test-auth', async (req, res) => {
+    try {
+        const apiKey = process.env.PUBLIC_KEY;
+        const privateKey = process.env.PRIVATE_KEY;
+
+        const termlyTimestamp = getTermlyTimestamp();
+        const authHeader = createAuthHeader(apiKey, privateKey, termlyTimestamp, 'GET', '', '/v1/authn', '');
+        console.log(`Authorization Header: ${authHeader}`);
+
+        const response = await axios.get('https://api.usw2.staging.trmly.net/v1/authn', {
```
Comment (Gidgidonihah) [inline auth-examples/nodejs/server.js]: Shouldn't this be the prod URL? Or do we give them staging accounts?

**Review by Gidgidonihah — CHANGES_REQUESTED** [review]

File: auth-examples/nodejs/server.js (hunk 1 above)
Comment (jona-termly) [reply auth-examples/nodejs/server.js]: Right you are, slipped through in testing

File: auth-examples/.DS_Store
Comment (jona-termly) [reply auth-examples/.DS_Store:1]: My gitignore appears to have been... ignored. Moved all rules to the root gitignore and deleted these.

**Review by Gidgidonihah — APPROVED** [review]

**Pull request merged**